alf/experience_replayers/experience_replay.py

```diff
@@ -85,14 +92,20 @@ class OnetimeExperienceReplayer(ExperienceReplayer):
     only once.
 
     Example algorithms: IMPALA, PPO2
+
+    NOTE: this replayer can only be run in the eager mode, because
+    self._experience is updated by python assignment
     """
 
-    def __init__(self, experience_spec, batch_size):
+    def __init__(self, experience_spec, *args, **kwargs):
```

Comment:
`*args, **kwargs` are not used , can we avoid this ?

Reply:
The reason is that the replayer will be created by the base class OffPolicyDriver. For tf_agents buffer, there is an additional arg `batch_size`. This is not used by this class.

Reply:
perhaps  we can  init replay_buffer in sub classes instead of specifing a cls and init it in base class

Reply:
The problem is that each sub-class can potentially use any of the replayers. I've changed the class to a string.